CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -148,14 +148,15 @@ import Vision
         }
         
         self.maskPreviewView(viewToMask: blurView, maskRect: frame)
-        
-        let borderWidth = CGFloat(5.0)
-        let cornersView = CornerView(frame: frame, borderWidth: borderWidth)
-        cornersView.layer.cornerRadius = self.regionCornerRadius + borderWidth
-        cornersView.layer.masksToBounds = true
-
-        cornersView.drawCorners(cornerColor: UIColor.green)
-        self.previewView?.addSubview(cornersView)
+//        print("during: \(frame)")
```

Comment:
Let's remove this commented out code. It's in github so we can find it later if needed, no need to keep something around if we aren't going to use it.